ramls/loan-policy.json

```diff
@@ -109,6 +109,15 @@
               "type": "object",
               "$ref": "period.json",
               "description": "Recall return interval"
+            },
+            "alternateRecallReturn": {
+              "type": "boolean",
+              "description": "Allow recall return interval for overdue loans"
+            },
+            "alternateRecallReturnInterval": {
+              "type": "object",
+              "$ref": "period.json",
+              "description": "Alternate recall return interval for overdue loans"
```

Comment:
I'd personally keep only the period property and if it exists - it means that alternate recall return interval is true, otherwise - false. 

If we still need two properties, then we probably should add a validation so that `alternateRecallReturnInterval` is specified if the flag is true and is not specified if the flag is false.

What do you think?

Reply:
Makes sense. I suppose I was being too literal in following the mockups.

Reply:
There is a part of me that likes the combination of the two properties, it makes the choice as to whether an alternate was chosen explicit and (if the combined validation is provided) makes accidental configuration less likely.